blackbox_repository/blackbox_surrogate.py

```diff
@@ -128,15 +128,10 @@ def _objective_function(
             return dict(zip(self.objectives_names, prediction))
         else:
             # returns all fidelities
-            objectives_values = []
-            for fidelity in self.fidelity_values:
-                surrogate_input = configuration.copy()
-                surrogate_input.update({next(iter(self.fidelity_space.keys())): fidelity})
-                # converts the returned nd-array with shape (1, num_metrics) to the list of objectives values
-                prediction = self.surrogate_pipeline.predict(pd.DataFrame([surrogate_input])).reshape(-1).tolist()
-                objectives_values.append(prediction)
-
-            return np.stack(objectives_values)
+            surrogate_input_df = pd.DataFrame([surrogate_input] * len(self.fidelity_values))
```

Comment:
Would be good to have a comment which explains how this works here?

Reply:
Sure, sent a commit with such a comment. Can you approve again?